github-web-interface.md

```diff
@@ -0,0 +1,42 @@
+# The GitHub Web Interface
+
+This is a guide to help those unfamiliar with git contribute to the TIL repository.
```

Comment:
Suggestion: This guide is to help Non-Git friends to contribute to the TIL repository.